### ISSUE TITLE
Failure to create PVC

### ISSUE THREAD

**Issue by zzzhangqi** [kadalu/kadalu#364]
System environment：centos arm64，k8s-1.18
**First question:**
```shell
#  I create it this way
kubectl kadalu storage-add rainbond-storage-pool-4 \
--device master-01:/dev/vdb
```
master-01 is an existing node

pods:
```shell
$ kubectl get pod -n kadalu
NAME                                 READY   STATUS    RESTARTS   AGE
kadalu-csi-nodeplugin-ll78d          3/3     Running   0          43m
kadalu-csi-nodeplugin-p8bwl          3/3     Running   0          43m
kadalu-csi-nodeplugin-rv42r          3/3     Running   0          43m
kadalu-csi-nodeplugin-rwdjk          3/3     Running   0          43m
kadalu-csi-nodeplugin-vhc4f          3/3     Running   0          43m
kadalu-csi-nodeplugin-xcd7p          3/3     Running   0          43m
kadalu-csi-provisioner-0             4/4     Running   0          43m
operator-76b85b86b6-dtwds            1/1     Running   0          43m
server-rainbond-storage-pool-4-0-0   2/2     Running   0          11m
```
```shell
#error logs :

Events:
  Type     Reason                Age                   From                                                                  Message
  ----     ------                ----                  ----                                                                  -------
  Warning  ProvisioningFailed    112s                  kadalu_kadalu-csi-provisioner-0_2e0f6b97-789b-488d-a866-aa6682151aa4  failed to provision volume with StorageClass "kadalu.replica1": rpc error: code = DeadlineExceeded desc = context deadline exceeded
  Normal   Provisioning          111s (x2 over 4m52s)  kadalu_kadalu-csi-provisioner-0_2e0f6b97-789b-488d-a866-aa6682151aa4  External provisioner is provisioning volume for claim "default/webapp-pv"
  Normal   ExternalProvisioning  3s (x21 over 4m52s)   persistentvolume-controller                                           waiting for a volume to be created, either by external provisioner "kadalu" or manually created by system administrator
```

**Second question:**
```shell
kubectl kadalu storage-add rainbond-storage-pool-5 --path master-01:/root/data/

#error logs:
Events:
  Type     Reason       Age               From               Message
  ----     ------       ----              ----               -------
  Normal   Scheduled    33s               default-scheduler  Successfully assigned kadalu/server-rainbond-storage-pool-5-0-0 to rainbond-master-01
  Warning  FailedMount  2s (x7 over 33s)  kubelet            MountVolume.SetUp failed for volume "glusterfsd-mountdir" : hostPath type check failed: /root/data/ is not a directory

# ls /root/data  Is the directory that exists

```

**Comment by amarts** [comment]
Possible to have 2 reasons:

I see that you mentioned CentOS7, in which case, we expect a xfs version mismatch with our image, so please do mkfs.xfs on the device before hand itself.

On the root path too, we expect `-oprjquota` and extended attribute support. Please check logs of 'server-rainbond-storage-pool' and also check the logs of `kadalu-csi-provisioner-0`. That should have more information.

**Comment by zzzhangqi** [comment]
The first problem is that reinstallation is already possible to create PVC.

The second problem is still the error described above.
`kadalu-csi-provisioner-0` no error log

BUG：
`operator-rke-v0.7.2.yml`  The RKE environment is similar to this bug #239 

The `/opt/rke/var/lib/kubelet/pods` in the YML file is incorrect.

**Comment by amarts** [comment]
@zzzhangqi can you check if the issues fixed with `0.7.6` version? I will close it as RKE issue is specifically handled. Please reopen if you still find issues.
